pkg/prometheus/operator.go

```diff
@@ -96,8 +97,7 @@ func NewTLSAssetSecret(p monitoringv1.PrometheusInterface, labels map[string]str
 	}
 }
 
-// ValidateRemoteWriteSpec checks that mutually exclusive configurations are not
-// included in the Prometheus remoteWrite configuration section.
+// ValidateRemoteWriteSpec validates the RemoteWriteSpec.
```

Comment:
```suggestion
// ValidateRemoteWriteSpec checks that mutually exclusive configurations are not
// included in the Prometheus remoteWrite configuration section, while validating 
// RemoteWriteSpec child fields.
```

I'm not sure about this one, I think it makes sense to keep the previous comment and extend with the extra functionality... I struggling to find a good comment though 😅